src/Apps/Artist/artistRoutes.tsx

```diff
@@ -138,6 +138,7 @@ export const artistRoutes: AppRouteConfig[] = [
           query artistRoutes_WorksForSaleQuery(
             $artistID: String!
             $input: FilterArtworksInput
+            # TODO:(?) only request the artist series aggregation if user has the Unleash flag enabled?
```

Comment:
@mzikherman I feel like you might know the answer to this. Is it worthwhile, and if so... how exactly? 😅

Reply:
Technically we can do it, otherwise there's some slight performance penalty for calculating that aggregation when it's not being used - but likely its so minimal we don' t have to worry about it.

If we did want to do that, it would be something like only conditionally including the `ARTIST_SERIES` value in the `aggregations` array if the flag is enabled (where you've added it above - https://github.com/artsy/force/pull/13041/files#diff-5de59716720cb8022e2091d39c68c9d2a7640ec5c504ca54176621b77a70f2f2R21).

Reply:
Cool, that's nice and simple 👍🏽 (I thought I recalled some more tricky stuff with graphql directives)

Will add a commit for that.

Reply:
Errm, ok not so simple. You can [access Unleash flags from the `onServerSideRender` function](https://github.com/artsy/force/blob/823985ce73398aa11a684293e8072d76dc938798/docs/unleash_a-b_testing.md?plain=1#L124-L142), which has access to feature flags via sharify via the server's response object.

But I can't see how to get that from within the `prepareVariables` function.

Reply:
Alright gonna leave this for now, on the assumption that aggregation only introduces a small penalty. I'll leave the comment in place as a little nudge/reminder.

Reply:
Re: GraphQL directives, you're probably thinking about occasions when an entire fragment would want to be conditionally excluded, for those we use a directive, namely something like: `@include(if: $shouldFetchFragment)`.

Reply:
Re: `prepareVariables`, I believe unleash stuff gets injected into sharify via a feature flag middleware - https://github.com/artsy/force/blob/8de9660f609eff162a1d1fb81bfc41a0624caacd/src/Server/middleware/featureFlagMiddleware.ts#L46

and i think that data should then be accessible inside `prepareVariables` by using the `getENV` helper - https://github.com/artsy/force/blob/8de9660f609eff162a1d1fb81bfc41a0624caacd/src/Utils/getENV.tsx

Reply:
Oh right, the local storage business. I think I've got it now in https://github.com/artsy/force/pull/13041/commits/b30dc348a122ec7063a9bb23a8b9b539f51c5c39 🙏🏽